app/javascript/packs/application.js

```diff
@@ -18,3 +18,5 @@ require("channels")
 //
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
+import "bootstrap"
+import "../stylesheets/application"
```

Comment:
Should this be "./stylesheets/application"

Reply:
EAGLE EYE